tests/test_scdip.py

```diff
@@ -201,6 +201,18 @@ def fill_journey_input(self, value=None):
             raise TypeError(f"Expected value argument of type list or NoneType, not {type(value)}")
         return
 
+    def fill_dialog(self, value=None):
+        dialog = WebDriverWait(self.browser, 10).until(
+            EC.presence_of_element_located((By.CSS_SELECTOR,'[role=document] .v-dialog--active')),
+            'Failed to locate restart dialog'
+        )
+        action_button = dialog.find_elements_by_css_selector('button')[0]
```

Comment:
This will need to be amended to allow selecting buttons other than the first one